From ec084b39099d2ea43a736ac46caffad90272ef9d Mon Sep 17 00:00:00 2001
From: Raymond Wagner <rwagner@mythtv.org>
Date: Wed, 16 Apr 2014 04:35:43 -0400
Subject: [PATCH] Remove file check for pytmdb3.cache

This is a partial roll back of 2c50a824b and remove the check for the
pytmdb3.cache file, allowing the library to simply create the file if
necessary.

Fixes #11876
---
 mythtv/programs/scripts/metadata/Movie/tmdb3.py |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/mythtv/programs/scripts/metadata/Movie/tmdb3.py b/mythtv/programs/scripts/metadata/Movie/tmdb3.py
index d4925e0..a442ad1 100755
--- a/mythtv/programs/scripts/metadata/Movie/tmdb3.py
+++ b/mythtv/programs/scripts/metadata/Movie/tmdb3.py
@@ -279,10 +279,7 @@ def main():
                 print "Unable to find MythTV directory for metadata cache."
                 sys.exit(1)
             confdir = os.path.join(confdir, '.mythtv')
-        confpath = os.path.join(confdir, 'cache/pytmdb3.cache')
-        if not os.access(confpath, os.F_OK|os.W_OK|os.R_OK):
-            print "Unable to access cache file: "+confpath
-            sys.exit(1)
+        confpath = os.path.join(confdir, 'cache', 'pytmdb3.cache')
         set_cache(engine='file', filename=confpath)
 
     if opts.language:
-- 
1.7.10.2

